public/theme/skeleton/search.twig

```diff
@@ -47,16 +47,14 @@
             {% endif %}
 
         {% else %}
-            <article>
 
                 <p>
-                    {{ __('general.phrase.no-result-for-search-term', { '%SEARCHTERM%': search|escape }) }}
+                    {{ __('general.phrase.no-search-results-for', { '%search%': search|escape }) }}
                 </p>
 
-            </article>
         {% endfor %}
 
         {# If there are more records than will fit on one page, the pager is shown. #}
-        {{ pager(template = 'partials/_sub_pager.twig') }}
+        {{ pager(records, template = 'partials/_sub_pager.twig') }}
```

Comment:
this template is default so can be omitted here

Reply:
In the past i've experienced that it's helpful for people who want to tweak it, to have this explicit. We've had a _lot_ of people asking "How do I change the pager template?"